src/components/Header/_NavLink.js

```diff
@@ -2,10 +2,11 @@ import React from "react";
 
 const NavLink = ({ Name, onClick, Active }) => {
   return (
-    <li className={"nav-item" + (Active ? " active" : "")}>
-      <span className="nav-link" onClick={onClick}>
-        {Name}
-      </span>
+      <li className={"nav-item" + (Active ? " active" : "")}>
+          <span className="nav-link">
+      <a  onClick={onClick}>
+                  {Name}
+              </a></span>
```

Comment:
This one's invalid and also can we follow the proper indenting guidelines, please. 😊